jupyter/pytorch+llmcompressor/test/test_notebook.ipynb

```
@@ -0,0 +1,80 @@
+    "            targets=\"Linear\",\n",
+    "            scheme=\"FP8_DYNAMIC\",\n",
+    "            ignore=[\"lm_head\"],\n",
+    "        )\n",
+    "\n",
+    "        # Apply quantization\n",
+    "        oneshot(\n",
+    "            model=model,\n",
+    "            recipe=recipe,\n",
+    "        )\n",
+    "\n",
+    "        # Save to disk in compressed-tensors format\n",
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)